tests/e2e/flowtest/test_scc.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import create_resource, wait_for_resource_state
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import Pod, get_pods_having_label
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.framework.pytest_customization.marks import bugzilla, polarion_id
+
+logger = logging.getLogger(__name__)
+
+
+class TestSCC:
+    @pytest.fixture(scope="function")
+    def create_scc(self, request):
+        scc_data = templating.load_yaml(constants.SCC_YAML)
+        scc_data["metadata"]["name"] = "scc-temp"
+        scc_obj = create_resource(**scc_data)
+        logger.info("New scc-temp is created")
+
+        def teardown():
+            scc_obj.delete()
+            logger.info("scc-temp is deleted")
+
+        request.addfinalizer(teardown)
+        return scc_obj
+
+    @bugzilla("1938647")
+    @polarion_id("OCS-4483")
+    def test_custom_scc(self, create_scc):
+        """
+        Test if OCS deployments/pods get affected if custom scc is created
+        """
+        # Add new scc to system:authenticated
+        OCP().exec_oc_cmd(
+            command=f"adm policy add-scc-to-group {create_scc.name} system:authenticated"
+        )
+
+        # Delete nooba-db pod
```

Comment:
I think it would be better to cover even other odf related pods

Reply:
we can discuss this offline